testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8 import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        
+        self.initial_car_order = [1, 2, 0, 3]
+        self.final_car_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(list(self.initial_car_order), list(self.final_car_order))

```

Comment:
@luxiya01 Do you afraid that is self.initial_car_order changes, then the value of the object changes? 
This will not happen e.g. I tested as shown below :)

```
def setUp(self):

        '''Basic setting that will be used in several tests
        '''
        
        self.initial_car_order = [1, 2, 0, 3]
        self.final_car_order = [3, 1, 2, 0]
        self.rc = RearrangeCars(self.initial_car_order, self.final_car_order)
        
    def test_change_init_order(self):
       print 'change the reference variable'
       self.initial_car_order = [3, 1, 2, 0]
       self.assertEqual(self.rc.initial_car_order, [1, 2, 0, 3])
```